webpack/components/AnsibleHostDetail/components/JobsTab/JobsTabHelper.js

```diff
@@ -7,21 +7,33 @@ import { showToast } from '../../../../toastHelper';
 export const ansiblePurpose = (resourceName, resourceId) =>
   `ansible-${resourceName}-${resourceId}`;
 
-const jobSearch = (resourceName, resourceId, statusSearch) =>
-  `recurring = true && pattern_template_name = "Ansible Roles - Ansible Default" && ${statusSearch} && recurring_logic.purpose = ${ansiblePurpose(
+const jobSearch = (resourceName, resourceId, status, hostGroupId) => {
+  const search = `recurring = true && pattern_template_name = "Ansible Roles - Ansible Default"`;
+  const searchStatus = ` && ${status}`;
+  const searchHost = ` && recurring_logic.purpose = ${ansiblePurpose(
     resourceName,
     resourceId
   )}`;
+  const searchHostGroup = hostGroupId
+    ? ` or recurring_logic.purpose = "ansible-hostgroup-${hostGroupId}"`
```

Comment:
Can we use the helper? IMHO it would help us flag places that needs to be changed if we change the purpose in the future.
```suggestion
    ? ` or recurring_logic.purpose = "ansible-ansiblePurpose('hostgroup', hostGroupId)"
```